### ISSUE TITLE
using it produces a script 1002 syntax error in Internet Explorer 11

### ISSUE THREAD

**Issue by cristianbell** [XiongAmao/vue-easy-lightbox#46]
We are using the library in our project. All nice and fine in Chrome, Firefox but produces a `script 1002 syntax error` in Internet Explorer 11. 

We are not doing anything special, just using `import VueEasyLightbox` and then using it in a component. 

Anyone else has encountered this issue? Any ideas? 

Note: this happens for all versions: 11, 12.1 and 13.

**Comment by XiongAmao** [comment]
Hi, if you are using vue-cli, you can refer to this config [vue.config.js transpiledependencies](https://cli.vuejs.org/config/#transpiledependencies).
Add `vue-easy-lightbox` to this option, like this:
```js
module.exports = {
  transpileDependencies: [
    'vue-easy-lightbox'
  ]
}
```

Or add `vue-easy-lightbox` to the `include` option of the webpack `rules` that include the `babel-loader`.

```js
module: {
  rules: [
    {
      test: /\.m?js$/,
      include: /vue-easy-lightbox/,
      exclude: /(node_modules/,
      use: {
        loader: 'babel-loader'
      }
    }
  ]
}
```
Normally,  `import 'vue-easy-lightbox/dist/vue-easy-lightbox.es5.esm.min.js'` can solve this problem.
But after I tested it, the dist file doesn't seem to compile completely to es5.
I will try to fix it later.

**Comment by XiongAmao** [comment]
@cristianbell 
Hi, this issue has been fixed in v0.14.0.
No need to use the solutions above.

**Comment by cristianbell** [comment]
Hi! 
Thanks for fixing it, I wanted to reply to your first post but was busy with other work. 

Cheers and thanks again!
